### DISCUSSION TITLE
Alert configuration via Dashboard (similar to the automation editor)

### DISCUSSION THREAD

**Discussion by redice2k** [home-assistant/frontend#14800] [Other feature requests] [answered]
I'm a big fan of alerts, but having to configure them all the time via yaml and having to restart HA to make them active is not very exciting :).
You've done an amazing job with the automation and script editor (and thank you very much for that), making it easier for everyone to create automations or scripts.
I believe that Alerts, in some circumstances, are much more useful than creating an automation, an example for me is the management of sensor batteries; if the battery of a certain sensor goes below 10% an alert is triggered which sends me a notification every few minutes/hours, in this way I'm sure I don't forget to change it.

I was wondering if it was possible to create an editor similar to the automation for Alerts too.

Thanks and congratulations for the excellent work

**Comment by frenck** [comment, marked as answer]
This is not a frontend feature request, but a backend one. Please use the community forums to request such features.

Thanks 👍 

../Frenck

**Reply by redice2k** [reply]
ok sorry, do I have to close this request or does it close automatically?

**Reply by frenck** [reply]
Its ok as it is right now 👍

**Comment by ttreker** [comment]
So to clarify, frenck, posting feature requests here is about the frontend only as it relates to general features of the core **Home Assistant** frontend or general UI elements that core **Home Assistant** provides for _Integrations_ and _Addons_ to use, but if there is a specific _Integration_ or _Addon_ that provides specific configuration or control interfaces via these **Home Assistant** features, even though these interfaces are UI components they should be discussed in the _Device and Service Support_ feature request forum?

I ask this because the _Feature Request Guidelines_ appear to suggest differently in my reading of them. Some clarification would be helpful here.

**Reply by frenck** [reply]
Please use our community forums. Thanks! 👍
